roles/st2mistral/tasks/main.yml

```diff
@@ -1,18 +1,27 @@
 ---
+- name: Install st2mistral dependency
+  become: yes
+  package:
+    name: st2python
```

Comment:
Created an Issue for that: https://github.com/StackStorm/st2-packages/issues/407 so we won't forget about it.